Spigot-Server-Patches/0151-Add-UnknownCommandEvent.patch

```diff
@@ -5,35 +5,27 @@ Subject: [PATCH] Add UnknownCommandEvent
 
 
 diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
-index 5d357b0f84b5242066dcce203752a0f46e9a249c..4f9c42a4b0256f181263bf5e0492714a01fbec38 100644
+index 268d56859a0de3b2ca39155f882991b5bf3aa5e0..54b68bad12f22d793e02f4583d7ab12ba3c54dc6 100644
 --- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
 +++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
-@@ -104,6 +104,7 @@ import net.minecraft.world.level.storage.WorldNBTStorage;
- import net.minecraft.world.level.storage.loot.LootTableRegistry;
- import net.minecraft.world.phys.Vec3D;
- import org.apache.commons.lang.Validate;
-+import org.apache.commons.lang3.StringUtils;
```

Comment:
Don't remove imports.

Reply:
I removed it because it was an unused import.
rolled back again!

Reply:
Hmm... it looks like this import actually should be removed. It's added by the patch which is being modified here, and is no longer in use due to changes in this PR.